### ISSUE TITLE
Sender name does not update on announcements

### ISSUE THREAD

**Issue by georgemblack** [Chaseshak/BadgerNet#88]
On the announcements list, the sender's name is not updated when that user changes his/her name on the settings page.

**Comment by rbrezynski** [comment]
Yeah right now sender name is just saved as a String when the announcement is sent. But we can discuss if we want to change this fuctionality
